Snakefile

```diff
@@ -43,14 +43,14 @@ SECDIR_path = pathlib.Path(
     "workflow",
     "pypsa-earth",
     "resources",
-    run_name,
+    SECDIR,
```

Comment:
As a refactoring comment, `SECDIR` may be is misleading and better can be revised into something styled similarly to `run_name`